dokka-subprojects/plugin-base/src/main/resources/dokka/styles/style.css

```diff
@@ -686,6 +686,24 @@ code.paragraph {
     text-decoration: line-through;
 }
 
+@-moz-document url-prefix() {
+    .strikethrough {
+        position: relative;
+        text-decoration: none;
+    }
+
+    .strikethrough::after, .strikethrough span:after  {
```

Comment:
Selectors in the solution are a bit complex but it is required to handle multiline cases

Reply:
Is it worth leaving this message as a code comment? Together with a link to the issue that was fixed, for future generations who might try to optimize / refactor it :sweat_smile: